data-products/src/main/scala/org/sunbird/analytics/job/report/AssessmentMetricsJobV2.scala

```diff
@@ -173,20 +173,28 @@ object AssessmentMetricsJobV2 extends optional.Application with IJob with BaseRe
         col("username"),
         col(UserCache.userchannel))
 
+    val userDenormDF = if (!reportId.toLowerCase.equalsIgnoreCase("nishtha-reports")) {
```

Comment:
Don't put this condition based on report Id. it should be a new configuration property. May be `all_channel_data` true. The default value will be false.

Reply:
Done